src/main/java/com/liaison/service/datamapper/contivo/TransformUpdater.java

```
@@ -230,30 +206,32 @@ protected String determineTransformFileExtension(String projectName, String mapR
 	protected void updateTransformFileSet(String projectRef, String mapRef, String fileExtension, String commitID,
 			Map<String, ChangeEntry> changeEntries) {
 		logger.debug("Using transform file with extension = " + fileExtension);
-
-		File localTransformMapClassFile = new File(getContivoRuntimeClasspathDirectory(), mapRef + fileExtension);
-
-		if (!changeEntries.containsKey(constructUrl(projectRef, mapRef + fileExtension))) {
-			// Fetch the transformation maps from GitLab if not already fetched
-			fetchFileFromGitlab(projectRef, mapRef, fileExtension, commitID, changeEntries);
-		}
 		// preflight check: Sometimes master file is remote and inaccessible.
 		if (!isMasterFileDirAccessible()) {
-			if (localTransformMapClassFile.exists()) {
-				return; // End early if the master directory is inaccessible but
-						// the file already exists.
-			} else {

```

Removed if since this is just an updater and not part of the transform process, we want to throw an error if we cant update